src/routes/v1.ts

```diff
@@ -94,7 +94,10 @@ export default async function (
         );
         const translated = {
           ...result,
-          incentives: transformIncentives(result.incentives, language),
+          incentives: transformIncentives(
```

Comment:
nit:  the filtering happening nested here could be easy to overlook. Do you think it would be valuable to pull this up at at all? For example...
```javascript

const result = ... ;
const filteredIncentives = result.incentives.filter(i => i.eligible);
const translated = {...result, incentives: transformIncentives(filteredIncentives, langage) ...
```

It can sometimes be hard to grok when you have multiple nested operations.